### PR TITLE
Made some UI tweaks 

### PR DESCRIPTION
**Pull request by jonathan-best**
Cleaned up the markup a little and reproduced the two columns
Update the JS to use the same form select var
Updated the styling

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -1,73 +1,83 @@
 body {
-    background-image: url("../img/tile.jpg");
-    background-color: rgba(255, 255, 255, 1);
-    font-family: 'EB Garamond', serif;
-    font-size: 250%;
+  background-color: #e1e1e1;
+  font-family: 'Roboto', serif;
+  font-weight: 300;
+  font-size: 200%;
+  max-width: 1200px;
```
Comment (jspc) [inline css/main.css]: We need to build this into an actual panel, really, to make sure a lot of these sizes are correct

File: css/main.css [hunk 2]
```diff
@@ -1,73 +1,83 @@
...
+  margin: 0 auto;
 }
 
-.container-inner {
-    border: 1px solid black;
-    /* background-color: rgba(255, 255, 255, 1); */
+.container-fluid{
```
Comment (jspc) [inline css/main.css]: whitespace before the brace, no?

File: js/main.js [hunk 3]
```diff
@@ -175,38 +177,40 @@
...
+              link_4: {
+                  uri: $( "input[name=link4]", form ).val(),
+                  text: $( "input[name=link4text]", form ).val(),
+              }
+          };
+
+          var f = {
+              brand: $( "input[name=brand]", form ).val(),
+              detail: $( "textarea[name=detail]", form ).val(),
+              file: document.getElementById('asset').files[0],
```
Comment (jspc) [inline js/main.js:204]: Could you make these selectors jQuery also? Or switch the others? Its on my list to do but it'd be cool if you could bundle it into this

Comment (jonathan-best) [inline js/main.js:204]: This can be done in another tidy up task - ideally we should remove jQuery completely 👍

**Pull request merged**